lib/perl5db/t/gh-21350

```diff
@@ -0,0 +1,3 @@
+#!perl
+
+print "minimal program to allow the debugger to launch\n";
```

Comment:
New files should be added to `MANIFEST`.

Reply:
Definitely, missed that.

Thank you for the review! Very helpful.